Tests/RxCocoaTests/RxTest+Controls.swift

```diff
@@ -62,6 +62,57 @@ extension RxTest {
         XCTAssertTrue(comparer(initialValue, lastReturnedPropertyValue), "last property value (\(lastReturnedPropertyValue)) does not match initial value (\(initialValue))", file: file, line: line)
     }
 
+    func ensurePropertyDeallocated<C, T: Equatable>(_ createControl: () -> C,
+                                                    _ initialValue: T,
+                                                    timeout seconds: TimeInterval,
+                                                    file: StaticString = #file,
+                                                    line: UInt = #line,
+                                                    _ propertySelector: (C) -> ControlProperty<T>) where C: NSObject {
+
+        ensurePropertyDeallocated(createControl, initialValue, timeout: seconds, comparer: ==, file: file, line: line, propertySelector)
+    }
+
+    func ensurePropertyDeallocated<C, T>(_ createControl: () -> C,
+                                         _ initialValue: T,
+                                         timeout seconds: TimeInterval,
+                                         comparer: (T, T) -> Bool,
+                                         file: StaticString = #file,
+                                         line: UInt = #line,
+                                         _ propertySelector: (C) -> ControlProperty<T>) where C: NSObject  {
+
+        let variable = Variable(initialValue)
+
+        let completeExpectation = XCTestExpectation(description: "completion")
+        let deallocateExpectation = XCTestExpectation(description: "deallocation")
+        var lastReturnedPropertyValue: T!
+
+        autoreleasepool {
+            var control: C! = createControl()
+
+            let property = propertySelector(control)
+
+            let disposable = variable.asObservable().bind(to: property)
+
+            _ = property.subscribe(onNext: { n in
+                lastReturnedPropertyValue = n
+            }, onCompleted: {
+                completeExpectation.fulfill()
+                disposable.dispose()
+            })
+
+
+            _ = (control as NSObject).rx.deallocated.subscribe(onNext: { _ in
+                deallocateExpectation.fulfill()
+            })
+
+            control = nil
+        }
+
+        wait(for: [completeExpectation, deallocateExpectation], timeout: seconds, enforceOrder: true)
```

Comment:
I think that we can modify original implementation of `func ensurePropertyDeallocated` and add `timeout seconds: TimeInterval = 0.0` and the following line to it.

` wait(for: [completeExpectation, deallocateExpectation], timeout: seconds, enforceOrder: true)`

Reply:
Seems that `wait(for:timeout:enforceOrder:)` function doesn't have `file` and `line` parameters.

Reply:
> Seems that wait(for:timeout:enforceOrder:) function doesn't have file and line parameters.

Maybe I'm overlooking something, why is this important?